### PR TITLE
docs: typos

### PR DESCRIPTION
**Pull request by Thoughtscript**
### Description

Some small changes to documentation to get familiar with CI/CD, tooling, etc. 

Thanks!

### Issue

1. Some nitpicks/fairly trivial typos fixed.
2. Was asked to separate these changes from another PR: https://github.com/kubernetes/website/pull/48622#pullrequestreview-2421767186 with suggestion from: @ArvindParekh https://github.com/kubernetes/website/pull/48622#discussion_r1826954596

@sftim - Here are the English content changes per your request! Thanks!

### CODE REVIEW

**Review by ArvindParekh — APPROVED** [review]
Thanks!
/lgtm

File: content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ a simple nginx web server to demonstrate the concept.
...
 
-The following is the example of the flow described in the
+The following is the example flow described in the 
 [Termination of Pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination)
 document.
 
-Let's say you have a Deployment containing of a single `nginx` replica
-(just for demonstration purposes) and a Service:
+Let's say you have a Deployment containing a single `nginx` replica
+(say just for the sake of demonstration purposes) and a Service:
```
Comment (utkarsh-singh1) [inline content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md:37]: The previous statement was better suited as new one does not add any new changes.

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ kubectl expose deployment/my-nginx
 service/my-nginx exposed
 ```
 
-This is equivalent to `kubectl apply -f` the following yaml:
+This is equivalent to `kubectl apply -f` in the following yaml:
```
Comment (stmcginnis) [inline content/en/docs/tutorials/services/connect-applications-service.md:93]: That command is not "in" the yaml file. The original statement is correct, but if you want to clarify, maybe something like the following would be better?

```txt
This is equivalent to running `kubectl apply -f` to a file containing the following yaml:
```

File: content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md (hunk 1 above)
Comment (stmcginnis) [reply content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md:37]: Agreed

**Review by stmcginnis — COMMENTED** [review]
Some decent minor improvements, but a couple small things pointed out.

File: content/en/docs/tutorials/services/connect-applications-service.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/tutorials/services/connect-applications-service.md:93]: We actually call them "manifests", not YAMLs

**Review by sftim — COMMENTED** [review]
Some positive changes, some not so helpful.

Anyway let's merge

/approve

**Pull request merged**